src/index.html

```diff
@@ -7,8 +7,47 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="link__moyo">
+        <img
+          src="images/logo.png"
+          alt="logoMOYO"
+          class="logo__moyo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="list__wrapper">
+          <li class="head__list">
+            <a class="head__link is-active" href="#">Apple</a>
```

Comment:
Тобі непотрібно придумувати велосипед
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a class="nav__link is-active" href="#">Apple</a>
```